src/map/entities/mobentity.cpp

```diff
@@ -1105,6 +1105,10 @@ void CMobEntity::OnEngage(CAttackState& state)
             PPet    = state.GetTarget();
             PTarget = ((CPetEntity*)PTarget)->PMaster;
         }
+
+        // TODO: Supertanking might be effected by this block when we don't want it to be.
+        // Things like Ambuscade "don't have" supertanking, though.
+        // This block apparently only effects rare things like NW apollyon, so might be ok for now.
```

Comment:
So this is a pretty bad implementation of alliance hate, where mobs will aggro the rest of the party if its target dies even if they haven't acted at all. I think an example of something that has this _and_ supertanking is Divine Might. The implementation is bad and will need to be changed, but there should be a way to add a player to a mobs "I'm gonna mess you up when I'm done with this guy" list while still allowing supertanking.

Reply:
In LSB this mod is only used in Apollyon NW on the last floor

Reply:
It never got widely implemented, mostly because it wasn't a great implementation but worked well enough in Limbus. I did think it was being used in more than just the 1 battlefield though.

Reply:
Hmm. Maybe I didn't look hard enough?

Reply:
It looks like it was part of the battlefield mob setup functions in mobutils but was apparently removed probably when that Limbus rewrite went in. I'm surprised NW is still left over. Either way there should exist a way for a mob to have a backup list of targets while still allowing supertanking. Not making any request here and if it was up to me I'd say just rip out any remnants of the existing alli_hate_mobmod. It's kind of obscure anyway and having working supertanking is more important right now.

Reply:
There's plans in the works to get better enmity systems, stay tuned

Reply:
Yeah so I looked and this mod isn't even being used uniformly for all the mobs in the zone, just the last floor in NW and the first floor in NE. Weird, it should all just be removed.